src/config.rs

```diff
@@ -2829,6 +2829,26 @@ impl TiKvConfig {
             );
         }
 
+        if (self.server.max_grpc_send_msg_len as u64)
+            < self.raft_store.raft_entry_max_size.0
+                + self.server.raft_client_grpc_send_msg_buffer as u64
+        {
+            warn!(
+                "server.max-grpc-send-msg-len {} < server.raft-client-grpc-send-msg-buffer {} + raftstore.raft-entry-max-size {},
+                 reset server.max-grpc-send-msg-len to {} = {} + {}",
+                self.server.max_grpc_send_msg_len,
+                self.server.raft_client_grpc_send_msg_buffer,
+                self.raft_store.raft_entry_max_size.0,
+                self.server.raft_client_grpc_send_msg_buffer as u64
+                    + self.raft_store.raft_entry_max_size.0,
+                self.server.raft_client_grpc_send_msg_buffer,
+                self.raft_store.raft_entry_max_size.0
+            );
+            self.server.max_grpc_send_msg_len =
```

Comment:
I don't think it's a good idea. The value is also used as the batch size. It's meaningless to batch every messages up to hundreds of megabytes. Better decouple them.